mongo/src/main/java/com/redhat/lightblue/mongo/crud/DocVerUtil.java

```diff
@@ -0,0 +1,119 @@
+/*
+ Copyright 2013 Red Hat, Inc. and/or its affiliates.
+
+ This file is part of lightblue.
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.redhat.lightblue.mongo.crud;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Date;
+
+import com.mongodb.DBObject;
+import com.mongodb.BasicDBObject;
+
+import org.bson.types.ObjectId;
+
+public class DocVerUtil {
+
+    public static final String DOCVER="docver";
+
+    public static final long TOO_OLD_MS=1l*60l*1000l; // Any docver older than 1 minute is to old
```

Comment:
Can I do this, as Lightblue client?
1. Read entity
2. Make changes
3. Save entity back to Lightblue, knowing that it will err if the entity was modified in the mean time (it's docver changed).

Considering this, when is the docver created initially?